ui/src/shared/components/RangeThresholdMarkers.tsx

```diff
@@ -0,0 +1,66 @@
+// Libraries
+import React, {FunctionComponent} from 'react'
+import {Scale} from '@influxdata/giraffe'
+
+// Components
+import ThresholdMarker from 'src/shared/components/ThresholdMarker'
+import ThresholdMarkerArea from 'src/shared/components/ThresholdMarkerArea'
+
+// Utils
+import {isInDomain, clamp} from 'src/shared/utils/vis'
+import {DragEvent} from 'src/shared/utils/useDragBehavior'
+
+// Types
+import {RangeThresholdConfig} from 'src/types'
+
+interface Props {
+  yScale: Scale<number, number>
+  yDomain: number[]
+  threshold: RangeThresholdConfig
+  onChangeMaxPos: (e: DragEvent) => void
+  onChangeMinPos: (e: DragEvent) => void
+}
+
+const RangeThresholdMarkers: FunctionComponent<Props> = ({
+  yScale,
+  yDomain,
+  threshold: {level, within, minValue, maxValue},
+  onChangeMinPos,
+  onChangeMaxPos,
+}) => {
+  const minY = clamp(yScale(minValue), yDomain)
+  const maxY = clamp(yScale(maxValue), yDomain)
```

Comment:
oops, just realized this should be 
```suggestion
  const maxY = yScale(clamp(maxValue, yDomain))
```

will fix before merging